src/misc/dbg_helpers.h

```diff
@@ -67,28 +66,32 @@ inline typename ArrayT<T>::item_t ItemAtT(E idx, const T &t, typename ArrayT<T>:
  * or t_unk when index is out of bounds.
  */
 template <typename E, typename T>
-inline CStrA ComposeNameT(E value, T &t, const char *t_unk, E val_inv, const char *name_inv)
+inline std::string ComposeNameT(E value, T &t, const char *t_unk, E val_inv, const char *name_inv)
 {
-	CStrA out;
+	std::string out;
 	if (value == val_inv) {
 		out = name_inv;
 	} else if (value == 0) {
 		out = "<none>";
 	} else {
 		for (size_t i = 0; i < ArrayT<T>::length; i++) {
 			if ((value & (1 << i)) == 0) continue;
-			out.AddFormat("%s%s", (out.Size() > 0 ? "+" : ""), (const char*)t[i]);
+			out += (!out.empty() ? "+" : "");
+			out += t[i];
 			value &= ~(E)(1 << i);
 		}
-		if (value != 0) out.AddFormat("%s%s", (out.Size() > 0 ? "+" : ""), t_unk);
+		if (value != 0) {
+			out += (!out.empty() > 0 ? "+" : "");
```

Comment:
Funny how MSVC annotates each time the function is used, but the warning is very valid `!out.empty() > 0` definitely looks wrong.